tools/localization/move_uwp_resources.ps1

```diff
@@ -1,67 +0,0 @@
-# List of resource folders
```

Comment:
Me deleting this file is probably the reason the "Legacy" build is failing, FYI. You can restore it if you want.

Reply:
This moves the UWP resources from the old cdpx localization names to the preferred UWP folders.

My new script (move-uwp-resw) handles the same thing for the Touchdown names.